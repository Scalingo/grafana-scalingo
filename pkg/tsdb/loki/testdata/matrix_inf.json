{
  "status": "success",
  "data": {
    "resultType": "matrix",
    "result": [
      {
        "metric": {
          "level": "info",
          "location": "moon",
          "protocol": "http"
        },
        "values": [
          [1643021611.9, "+Inf"],
          [1643021911.9, "-Inf"]
        ]
      }
    ]
  }
}
