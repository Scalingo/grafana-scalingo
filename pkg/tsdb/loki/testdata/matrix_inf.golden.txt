ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "executedQueryString": "Expr: up(ALERTS)\nStep: 42s"
}
Name: {level="info", location="moon", protocol="http"}
Dimensions: 2 Fields by 2 Rows
+---------------------------------+--------------------------------------------------+
| Name: time                      | Name: value                                      |
| Labels:                         | Labels: level=info, location=moon, protocol=http |
| Type: []time.Time               | Type: []float64                                  |
+---------------------------------+--------------------------------------------------+
| 2022-01-24 10:53:31.9 +0000 UTC | +Inf                                             |
| 2022-01-24 10:58:31.9 +0000 UTC | -Inf                                             |
+---------------------------------+--------------------------------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////6AIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAANgAAAADAAAAfAAAACgAAAAEAAAAtP3//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADU/f//CAAAADwAAAAwAAAAe2xldmVsPSJpbmZvIiwgbG9jYXRpb249Im1vb24iLCBwcm90b2NvbD0iaHR0cCJ9AAAAAAQAAABuYW1lAAAAACT+//8IAAAAQAAAADUAAAB7ImV4ZWN1dGVkUXVlcnlTdHJpbmciOiJFeHByOiB1cChBTEVSVFMpXG5TdGVwOiA0MnMifQAAAAQAAABtZXRhAAAAAAIAAAA8AQAABAAAAN7+//8UAAAABAEAAAQBAAAAAAADBAEAAAMAAACEAAAALAAAAAQAAACs/v//CAAAABAAAAAFAAAAdmFsdWUAAAAEAAAAbmFtZQAAAADQ/v//CAAAAEAAAAA0AAAAeyJsZXZlbCI6ImluZm8iLCJsb2NhdGlvbiI6Im1vb24iLCJwcm90b2NvbCI6Imh0dHAifQAAAAAGAAAAbGFiZWxzAAAk////CAAAAFgAAABOAAAAeyJkaXNwbGF5TmFtZUZyb21EUyI6IntsZXZlbD1cImluZm9cIiwgbG9jYXRpb249XCJtb29uXCIsIHByb3RvY29sPVwiaHR0cFwifSJ9AAAGAAAAY29uZmlnAAAAAAAAVv///wAAAgAFAAAAdmFsdWUAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAAB4AAAAgAAAAAAAAAqAAAAAAgAAADQAAAAEAAAA3P///wgAAAAQAAAABAAAAHRpbWUAAAAABAAAAG5hbWUAAAAACAAMAAgABAAIAAAACAAAABwAAAASAAAAeyJpbnRlcnZhbCI6NDIwMDB9AAAGAAAAY29uZmlnAAAAAAAAAAAGAAgABgAGAAAAAAADAAQAAAB0aW1lAAAAAP////+4AAAAFAAAAAAAAAAMABYAFAATAAwABAAMAAAAIAAAAAAAAAAUAAAAAAAAAwQACgAYAAwACAAEAAoAAAAUAAAAWAAAAAIAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAAAAAAAIAAAACAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAADXlm9zL80WAI/7SLkvzRYAAAAAAADwfwAAAAAAAPD/EAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADwAAAAAAAQAAQAAAPgCAAAAAAAAwAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAACgAMAAAACAAEAAoAAAAIAAAA2AAAAAMAAAB8AAAAKAAAAAQAAAC0/f//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAANT9//8IAAAAPAAAADAAAAB7bGV2ZWw9ImluZm8iLCBsb2NhdGlvbj0ibW9vbiIsIHByb3RvY29sPSJodHRwIn0AAAAABAAAAG5hbWUAAAAAJP7//wgAAABAAAAANQAAAHsiZXhlY3V0ZWRRdWVyeVN0cmluZyI6IkV4cHI6IHVwKEFMRVJUUylcblN0ZXA6IDQycyJ9AAAABAAAAG1ldGEAAAAAAgAAADwBAAAEAAAA3v7//xQAAAAEAQAABAEAAAAAAAMEAQAAAwAAAIQAAAAsAAAABAAAAKz+//8IAAAAEAAAAAUAAAB2YWx1ZQAAAAQAAABuYW1lAAAAAND+//8IAAAAQAAAADQAAAB7ImxldmVsIjoiaW5mbyIsImxvY2F0aW9uIjoibW9vbiIsInByb3RvY29sIjoiaHR0cCJ9AAAAAAYAAABsYWJlbHMAACT///8IAAAAWAAAAE4AAAB7ImRpc3BsYXlOYW1lRnJvbURTIjoie2xldmVsPVwiaW5mb1wiLCBsb2NhdGlvbj1cIm1vb25cIiwgcHJvdG9jb2w9XCJodHRwXCJ9In0AAAYAAABjb25maWcAAAAAAABW////AAACAAUAAAB2YWx1ZQASABgAFAAAABMADAAAAAgABAASAAAAFAAAAHgAAACAAAAAAAAACoAAAAACAAAANAAAAAQAAADc////CAAAABAAAAAEAAAAdGltZQAAAAAEAAAAbmFtZQAAAAAIAAwACAAEAAgAAAAIAAAAHAAAABIAAAB7ImludGVydmFsIjo0MjAwMH0AAAYAAABjb25maWcAAAAAAAAAAAYACAAGAAYAAAAAAAMABAAAAHRpbWUAAAAAGAMAAEFSUk9XMQ==
